crates/mirror-mirror/src/type_info/mod.rs

```diff
@@ -841,8 +878,15 @@ impl<'a> EnumType<'a> {
 
     pub fn default_value(self) -> Option<Value> {
         let mut variants = self.variants();
-        let variant = variants.next()?;
-        variant.default_value()
+        let first_variant = variants.next()?;
+        first_variant.default_value()
+    }
+
+    pub fn has_default_value(&self) -> bool {
+        let mut variants = self.variants();
+        variants
+            .next()
+            .map_or(false, |first_variant| first_variant.has_default_value())
```

Comment:
Would probably be nice to try to iter the variants and find the first one that has a default value and then construct that, thoughts? Can be follow up work though

Reply:
Yeah I think that makes sense!